### ISSUE TITLE
Rails 7 support

### ISSUE THREAD

**Issue by kapso** [igorkasyanchuk/active_storage_validations#141]

**Comment by igorkasyanchuk** [comment]
It doesn't work? Can you make a fix?

**Comment by lavaturtle** [comment]
I'm trying to upgrade to Rails 7, and on my specs like this:
`it { is_expected.to validate_content_type_of(:image).allowing('image/jpeg', 'image/png', 'image/gif', 'image/svg+xml') }`
I get errors like this:
```
ActiveSupport::DeprecationException:
       DEPRECATION WARNING: image/bmp is not a valid content type, it should not be used when creating a blob, and support for it will be removed in Rails 7.1. If you want to keep supporting this content type past Rails 7.1, add it to `config.active_storage.variable_content_types`. Dismiss this warning by setting `config.active_storage.silence_invalid_content_types_warning = true`.
```

**Comment by kapso** [comment]
Sorry forgot to update, I was able to upgrade to Rails 7

**Comment by lavaturtle** [comment]
In my case, I worked around the issue by explicitly passing `.rejecting` to the matcher.
